documentation/testing/test-plans/20240711_release-test-plan.md

```diff
@@ -0,0 +1,81 @@
+# Release Test Plan
+Release date: 11/07/2024
+
+## Introduction
+Objective: The purpose of this test plan is to outline the approach and scope for testing the bug fixes made today. It will detail the various types of testing that will be conducted as part of release process.  This plan aims to ensure all aspects of the updates are thoroughly tested and meet the required standards before deployment.
```

Comment:
Suggest avoiding using `today`. Doesn't read right if you're reading this after the fact.

Reply:
good point. updated it now.